explorerlib/index.go

```diff
@@ -45,9 +46,12 @@ func (index *Index) FetchResponse(queryExpression string, numberOfResults int) (
 
 func (index *Index) BuildGoodQueries(wordCountsByLanguage map[string]WordCounts, numberOfQueryByLanguage int, averageNumberOfWords int) (map[string][]string, error) {
 	queriesInLanguage := make(map[string][]string)
+
+	tic := 0
 	for language, wordCounts := range wordCountsByLanguage {
 		words := []string{}
 		for i := 0; i < numberOfQueryByLanguage; {
+			fmt.Printf("\rBuilding %v queries", tic)
```

Comment:
Log progression by 1% increment: "Building and validating queries: x% completed"